patientsearch/config.py

```diff
@@ -14,11 +14,10 @@ def load_json_config(potential_json_string):
         return json.loads(potential_json_string)
 
 
-
 LANDING_INTRO = os.getenv("LANDING_INTRO", "")
 LANDING_BUTTON_TEXT = os.getenv("LANDING_BUTTON_TEXT", "")
 LANDING_BODY = os.getenv("LANDING_BODY", "")
-
+MORE_MENU = json.loads(os.getenv("MORE_MENU", '[]'))
```

Comment:
Can you use a comma-separated list here ([eg](https://github.com/uwcirg/truenth-portal/blob/develop/portal/config/config.py#L122-L123)), or do you expect deeper nested values?

Reply:
done.